alerter.c

```diff
@@ -1,32 +1,53 @@
 #include <stdio.h>
 #include <assert.h>
 
+#define Test_Code 1
+
 int alertFailureCount = 0;
 
+#ifdef Test_Code
 int networkAlertStub(float celcius) {
+    int returnValue = 0;
     printf("ALERT: Temperature is %.1f celcius.\n", celcius);
     // Return 200 for ok
     // Return 500 for not-ok
     // stub always succeeds and returns 200

```

Comment:
Comments are not valid anymore. Needs to be cleaned up before it creates confusion at a later date.